GitUI/UserControls/FileStatusList.cs

```diff
@@ -61,7 +62,12 @@ public sealed partial class FileStatusList : GitModuleControl
         [DefaultValue(false)]
         public bool DisableSubmoduleMenuItemBold { get; set; }
 
-        private readonly Dictionary<string, int> _stateImageIndexDict = [];
+        private static readonly ImageListData _imageListData;
+
+        static FileStatusList()
+        {
+            _imageListData = CreateImageListData();
+        }
```

Comment:
can be shortened to:
```suggestion
        private static readonly ImageListData _imageListData = CreateImageListData();
```